package.json

```diff
@@ -3,10 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev -p build-fragment",
-    "build": "next build -p build-fragment",
-    "start": "next start -p build-fragment",
-    "build-fragment": "node ./schemaQuery.ts"
+    "dev": "npm-run-all -p build-fragment dev-next",
+    "dev-next": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "build-fragment": "node ./schemaQuery.js"
```

Comment:
Change it back to `.js` as it will drive a lot of issues if this plain script file is written in `Typescript`.